### PR TITLE
feat: accept paths as an argument to -i flag (on top of folder and file name)

### PR DESCRIPTION
**Pull request by lsm-dev**
This PR implements feature #21 asked by @ericzon . 

The -i flag currently accepts a comma separated list of file/folder names.

Now it will be possible to provide paths too. It is backwards compatible with the previous solution.

### CODE REVIEW

File: internal/file/file.go [hunk 1]
```diff
@@ -45,8 +45,14 @@ func HasExtension(path string, extensions []string) bool {
 // ShouldIgnore returns true if the path matches any of the ignore strings
 func ShouldIgnore(path string, ignoreFolders []string) bool {
 	segments := strings.Split(path, string(os.PathSeparator))
-	for _, segment := range segments {
-		for _, ignore := range ignoreFolders {
+	for _, ignore := range ignoreFolders {
+		if strings.Contains(ignore, string(os.PathSeparator)) {
+			if strings.Contains(path, ignore) {
```
Comment (jcyamacho) [inline internal/file/file.go]: if **ignore** is: "ules/lab" then following paths will be ignored:
"modules/lab", "modules/laboratory/...", basically all paths matching: `/.*ules\/lab.*/`
chrome_HvrFHvs6T2

Comment (lsm-dev) [reply internal/file/file.go]: That is sooo true ;)

Adding a failing test in 3..2..1..

File: internal/file/file.go [hunk 2]
```diff
@@ -43,11 +44,14 @@ func HasExtension(path string, extensions []string) bool {
...
 	segments := strings.Split(path, string(os.PathSeparator))
-	for _, segment := range segments {
-		for _, ignore := range ignoreFolders {
-			if segment == ignore {
+	for _, ignore := range ignores {
+		sub := strings.Split(ignore, string(os.PathSeparator))
+		size := len(sub)
+		last := len(segments) - size
+		for i := 0; i <= last; i++ {
+			if reflect.DeepEqual(segments[i:i+size], sub) {
```
Comment (ericzon) [inline internal/file/file.go]: I suggest to be more descriptive (think in your future you): 
segments -> pathSegments
ignores -> pathsToIgnore
ignore -> pathToIgnore
sub -> ignorePathSegments
...

File: cmd/license-header-checker/main.go [hunk 3]
```diff
@@ -51,13 +51,14 @@ var (
 	errorRender   = color.FgRed.Render
 )
 
-func printIntro() {
-	fmt.Printf("\nFiles:\n")
+func printIntro(options *config.Options) {
+	if options.Verbose {
+		fmt.Printf("\nFiles:\n")
```
Comment (ericzon) [inline cmd/license-header-checker/main.go]: For next version, I suggest to delegate output format to helper

File: internal/file/file.go (hunk 2 above)
Comment (lsm-dev) [reply internal/file/file.go]: yep

**Pull request merged**